apps/mobile/index.js

```diff
@@ -1,10 +1,7 @@
 import registerRootComponent from 'expo/build/launch/registerRootComponent';
```

Comment:
The `index` file has to be `.js` or Android builds don't work. This doesn't super bother me but is weird.